app/config/ApplicationConfig.scala

```diff
@@ -24,11 +24,13 @@ import play.api.Mode.Mode
 import play.api.{Configuration, Environment, Play}
 import uk.gov.hmrc.play.config.{RunMode, ServicesConfig}
 
+import scala.util.Try
+
 @Singleton()
 class ApplicationConfig @Inject()(override val runModeConfiguration: Configuration, environment: Environment) extends ServicesConfig {
 
   private def loadConfig(key: String) = runModeConfiguration.getString(key).getOrElse(throw new Exception(s"Missing configuration key: $key"))
-  private def loadBooleanConfig(key: String) = runModeConfiguration.getBoolean(key).getOrElse(throw new Exception(s"Missing configuration key: $key"))
+  private def loadBooleanConfig(key: String) = Try(runModeConfiguration.getString(key).getOrElse(throw new Exception(s"Missing configuration key: $key")).toBoolean).getOrElse(false)
```

Comment:
I would do config.getConfString(key).fold(false)(_.toBoolean)

Reply:
OK cool

Reply:
changed to getString(key).fold(false)(_.toBoolean)